southwest_checkin.py

```diff
@@ -42,28 +43,52 @@
 # busy wait until it is the desired time
 des_time = datetime.strptime(options.time, '%b %d %Y %I:%M%p')
 des_time_minus_6s = des_time - timedelta(seconds=6)
+des_time_minus_30m = des_time - timedelta(minutes=30)
```

Comment:
I figured it'd be a good idea to add in a longer timeout as well, so that you could "set and forget" the script a day or two in advance.